perun-core/src/main/java/cz/metacentrum/perun/core/implApi/TasksManagerImplApi.java

```diff
@@ -116,4 +116,6 @@
 	 */
 	List<TaskResult> getTaskResultsForDestinations(List<String> destinationsNames) throws InternalErrorException;
 
+	List<Task> listAllSchedulableTasks();
```

Comment:
There is missing implementation for this method in ``TasksManagerImpl``.

Reply:
Fixed.